rpcwebsocket.go

```diff
@@ -681,22 +684,38 @@ func (m *wsNotificationManager) subscribedClients(tx *dcrutil.Tx, clients map[ch
 		}
 
 		for i, output := range msgTx.TxOut {
-			_, addrs, _, err := txscript.ExtractPkScriptAddrs(output.Version,
+			watchOutput := true
+			sc, addrs, _, err := txscript.ExtractPkScriptAddrs(output.Version,
 				output.PkScript, m.server.server.chainParams)
 			if err != nil {
 				// Clients are not able to subscribe to
 				// nonstandard or non-address outputs.
 				continue
 			}
+			if sc == txscript.NullDataTy {
+				// OP_RETURN ticket commitments may contain relevant
+				// P2PKH or P2SH HASH160s.
+				// These outputs cannot be spent and do not need to
+				// be watched.
+				addr, err := stake.AddrFromSStxPkScrCommitment(
```

Comment:
This function assumes that it is only being called with a commitment that has already been validated or it can return incorrect results.

I believe this should be limited to only apply to tickets and the specific commitment outputs.

Reply:
now only checks odd outputs of tickets.